### ISSUE TITLE
Support `CREATE TABLE` statements

### ISSUE THREAD

**Issue by Matts966** [minodisk/bigquery-runner#4]
Thank you for this great plugin!

Can a Pull request to support `CREATE TABLE` statements be accepted?

Current behavior returns empty data for `CREATE TABLE` statements and it would be great to check the results.

**Comment by Matts966** [comment]
@minodisk 
I will close this if not accepted :)

**Comment by minodisk** [comment]
@Matts966 
Thanks for using it.
I don't usually run `CREATE TABLE`, so I wasn't aware of its behavior.
I will check `CREATE TABLE` with the return value of the query and see how easy it is to support.

**Comment by Matts966** [comment]
You can get destination table references like

```ts
bigquery.createQueryJob(`
CREATE OR REPLACE TABLE \`test_expected.test\` AS
SELECT 1 as a;
`).then((job) => {
  console.log(job[1].configuration.query.destinationTable)
});

// returns

{
  projectId: 'project',
  datasetId: 'test_expected',
  tableId: 'test'
}

```

If this is accepted, I will try making a PR. If you have a plan to implement this, I will wait it :)

**Comment by minodisk** [comment]
I want you to try #6.

1. Pull the `create-table-as-select` branch to your local machine.
2. Run `npm i`
3. Run `npm run debug` to build BigQuery Runner and install into VSCode.
4. Run the command `Developer: Reload Window` in VSCode to activate the newly installed BigQuery Runner.
5. Run the `create or replace table` query with `BigQuery Runner: Run`.

https://user-images.githubusercontent.com/514164/156774998-ac5cda1a-d3d2-4848-a70e-763f1ab0b308.mov

**Comment by Matts966** [comment]
@minodisk 
Hi, this is exactly what we want! Could you please release this feature?

**Comment by minodisk** [comment]
@Matts966
Released as v0.0.36 😉

**Comment by minodisk** [comment]
https://marketplace.visualstudio.com/items/minodisk.bigquery-runner/changelog

**Comment by Matts966** [comment]
I found edge cases with characters such as `-` in the table identifiers, but it was already fixed in 0.0.37 :D

**Comment by Matts966** [comment]
The experience is great! I will share this with my colleagues :)
Thank you very much!
